src/ai/backend/common/lock.py

```diff
@@ -206,6 +211,11 @@ class RedisLock(AbstractDistributedLock):
     default_timeout = 9600
     default_lock_acquire_pause = 1.0
 
+    default_config: ClassVar[Mapping[str, Any]] = {"lock_acquire_pause": default_lock_acquire_pause}
+    config_iv: ClassVar[t.Trafaret] = t.Dict({
+        t.Key("lock_acquire_pause", default=None): t.Null | t.ToFloat[0:],
```

Comment:
I'd propose a better naming of this configuration option: `lock_retry_interval`.